src/__tests__/test-build.js

```diff
@@ -68,6 +72,8 @@ async function main () {
     assert.strict.equal(isVisible, true)
     console.log(chalk.green('✓'), 'app is visible')
 
+    await wait(2)
```

Comment:
Is it possible to do a wait for something to exist instead of a static timeout?

Reply:
Yes, I actually didn't mean to leave this in - will remove

Reply:
This is removed now